flow360/component/simulation/framework/cache_model_base.py

```diff
@@ -0,0 +1,20 @@
+import abc
+from typing import Any, Dict
+
+import pydantic as pd
+
+from flow360.component.simulation.framework.base_model import Flow360BaseModel
+
+
+class CacheModelBase(Flow360BaseModel, metaclass=abc.ABCMeta):
+    def __init__(self, **data):
+        cached = data.pop("_cached", None)
+        super().__init__(**data)
+        if cached:
+            self._cached = self.__annotations__["_cached"].model_validate(cached)
```

Comment:
It is going to be interesting when the `model_validate` gives error. Maybe we should catch this internally?